### ISSUE TITLE
WARNING! This repo seems to not be maintain anymore and contains many bugs. Think twice before using.

### ISSUE THREAD

**Issue by yarimiz** [sammchardy/python-binance#1260]
Hey there,

Before using this repo, please be aware of the following:

- This library wasn't updated since April 2022 although many bugs and PR were opened. (and all PR builds fails due to bad pipeline)
- There are a lot of bugs here and many places of half implemented methods. Naming convention is broken.
- Version 1.0.16 is broken, if you still want to use this library, use 1.0.15

I encourage you to read this before
https://github.com/sammchardy/python-binance/issues/1141

**Comment by jp-diegidio** [comment]
There exists by now an official Binance client (see binance/binance-connector-python), so there is no point in using this code anyway.

**Comment by yarimiz** [comment]
> There exists by now an official Binance client (see binance/binance-connector-python), so there is no point in using this code anyway.

Thanks, but there's still a reason because the official client does not support futures.

**Comment by jp-diegidio** [comment]
> > There exists by now an official Binance client (see binance/binance-connector-python), so there is no point in using this code anyway.
> 
> Thanks, but there's still a reason because the official client does not support futures.

Yes, the client for futures is this one: [binance/binance-futures-connector-python](https://github.com/binance/binance-futures-connector-python).

I have not investigated further (what is there, what is missing or somewhere else), but that is the future!, pun intended. :)  Indeed, a lot of stuff in binance's repositories, and a lot of (overall good, IMO) work going on...

Hence my advice not to waste any time with this one and just jump on that wagon.

**Comment by suchakpks** [comment]
I also faced issue with [binance/binance-futures-connector-python](https://github.com/binance/binance-futures-connector-python). this.
